### PR TITLE
Updated the lib URL of Sensirion UPT core

### PR DESCRIPTION
**Pull request by qfisch**
Updated the URL of Sensirion UPT core.
Thank you

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @qfisch. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2024/02/08 17:05:48 Loaded 6630 libraries from DB
Changing URL of library Sensirion_UPT_Core from https://github.com/Sensirion/upt-core.git to https://github.com/Sensirion/arduino-upt-core.git
Success!
```

**Pull request merged**
